src/mmrelay/meshtastic/plugins.py

```
@@ -0,0 +1,153 @@
+    """
+
+    raw_value = default
+    if isinstance(cfg, dict):
+        try:
+            raw_value = cfg.get("meshtastic", {}).get("plugin_timeout", default)
+        except AttributeError:
+            raw_value = default
+
+    try:
+        if isinstance(raw_value, bool):
+            raise TypeError("boolean timeout")
```

Ruff (0.15.9)

[warning] Abstract `raise` to an inner function

(TRY301)